src/main/java/io/specto/hoverfly/junit/core/model/GlobalActions.java

```diff
@@ -13,12 +13,16 @@
 package io.specto.hoverfly.junit.core.model;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
 
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I think this option should be baked into the POJO itself, otherwise, when anyone use this model, they will have to configure the objectMapper to configure this option. Plus configure() is an instance method, it just configures for one instance of objectMapper. You can do it globally in Spring is because you can inject it as a singleton bean. But I don't think we should expect the users of this library to do anything like that.